src/linear_algebra/matrix/determinant.lean

```diff
@@ -658,4 +650,33 @@ by { rw [← det_transpose, det_succ_row _ j],
      refine finset.sum_congr rfl (λ i _, _),
      rw [add_comm, ← det_transpose, transpose_apply, transpose_minor, transpose_transpose] }
 
+
+/-- Specialize `det_eq_one_of_card_eq_zero` to `fin 0`.
+
+This is especially useful in combination with the `det_succ_` lemmas,
+for computing the determinant of a matrix given in the `![...]` notation.
+-/
+@[simp] lemma det_fin_zero {A : matrix (fin 0) (fin 0) R}: det A = 1 :=
+det_eq_one_of_card_eq_zero (fintype.card_fin _)
+
+/-- Determinant of 1x1 matrix -/
+alias det_unique ← det_fin_one
```

Comment:
Making this an `alias` seems like a bad idea, as the resulting lemma doesn't actually end up specialized to `fin 1`